src/Caret.js

```diff
@@ -104,7 +101,7 @@ Caret.defaultProps = {
   borderColor: 'gray.2',
   borderWidth: 1,
   location: 'bottom',
-  size: 2,
+  size: 8,
```

Comment:
Can you provide some context on these changes?

Reply:
Looks we do a lot of math on the `size` prop to generate the triangle SVG so I figured it was easier to use a unitless pixel value. This seems like a special case to me. https://github.com/primer/components/blob/3ddb99a4c9537c40a63e2fa1535ac2f9a87df08f/src/Caret.js#L57-L72

Reply:
This feels a little risky to me - what if we add values to our space theme object and there is a value at the 8th index?